contracts/strategies/qv-hackathon/HackathonQVStrategy.sol

```diff
@@ -0,0 +1,269 @@
+        }
+        return hackersArray;
+    }
+
+    /// =========================
+    /// ==== EAS Functions =====
+    /// =========================
+
+    // Note: basic setup for using EAS
+    // Version: 0.27
+    // * OP Goerli
+    //    EAS Contract: 0xC2679fBD37d54388Ce493F1DB75320D236e1815e
+    //    Schema Registry: 0x0a7E2Ff54e76B8E6659aedc9103FB21c038050D0
+    // * Sepolia
+    //    EAS Contract: 0x1a5650d0ecbca349dd84bafa85790e3e6955eb84
+    //    Schema Registry: 0x7b24C7f8AF365B4E308b6acb0A7dfc85d034Cb3f
+
+    /// @dev Gets an attestation from the EAS contract.
+    /// @param uid The UUID of the attestation to get.
+    function getAttestation(bytes32 uid) public payable virtual returns (Attestation memory) {
+        Attestation memory attestation = eas.getAttestation(uid);
+        return attestation;
+    }
+
+    /// @dev Gets a schema from the SchemaRegistry contract.
+    /// @param uid The UID of the schema to get.
+    function getSchema(bytes32 uid) public payable virtual returns (SchemaRecord memory) {
+        SchemaRecord memory schemaRecord = schemaRegistry.getSchema(uid);
+        return schemaRecord;
+    }
+
+    function _getAllo() internal view returns (IAllo) {
+        return IAllo(allo);
+    }
+
+    /// @dev Adds a verifier to the list of authorized verifiers.
+    /// @param _verifier The address of the verifier to add.
+    function addVerifier(address _verifier) public onlyPoolManager(msg.sender) {
+        if (attestationSigners[_verifier]) {
+            revert ALREADY_ADDED();
+        }
+
+        attestationSigners[_verifier] = true;
+
+        emit VerifierAdded(_verifier);
+    }
+
+    /// @dev Removes a verifier from the list of authorized verifiers.
+    /// @param _verifier The address of the verifier to remove.
+    function removeVerifier(address _verifier) public onlyPoolManager(msg.sender) {
+        if (!attestationSigners[_verifier]) {
+            revert INVALID();
+        }
+
+        attestationSigners[_verifier] = false;
+
+        emit VerifierRemoved(_verifier);
+    }
+
+    /// @dev Sets the address of the EAS contract.
+    /// @param _easContractAddress The address of the EAS contract.
+    function setEASAddress(address _easContractAddress) public onlyPoolManager(msg.sender) {
+        eas = IEAS(_easContractAddress);
+
+        emit EASAddressUpdated(_easContractAddress);
+    }
+
+    /// @dev Submit an attestation request to the EAS contract.
+    /// @param _attestationRequestData An array of `AttestationRequestData` structures containing the attestation request data.
+    function submitAttestation(MultiAttestationRequest[] calldata _attestationRequestData)
+        public
+        payable
+        virtual
+        onlyPoolManager(msg.sender)
+        returns (bytes32[] memory)
+    {
+        if (!attestationSigners[msg.sender]) {
+            revert INVALID();
+        }
+
+        return eas.multiAttest(_attestationRequestData);
+    }
```

Comment:
function can be removed, or?
isn't it the supposed to be the same like `_registerRecipient` ?
by calling `bulkRegisterRecipient` on `Allo.sol` we have bulk attestations